firefox-suggest-history-vs-offline.toml

```diff
@@ -19,6 +19,18 @@ from_expression = """(
 window_start = -7
 window_end = 0
 
+[segments.searchsuggestion_users]
+select_expression = """
+  COALESCE((SUM(COALESCE((
+    SELECT SUM(x.value) FROM UNNEST(scalar_parent_urlbar_picked_searchsuggestion_sum
+  ) x), 0)) > 0), FALSE)
+"""
+data_source = "clients_daily"
+
+[segments.data_sources.clients_daily]
+window_start = -7
+window_end = 0
```

Comment:
Hi @scholtzan , I'm trying to create a new segment using clients_daily as a data source. I want to change the window_start and window_end parameters from their default, but I'm getting an error that the data source is missing a from expression. Shouldn't Jetstream already know what the clients daily data source is? Is there a better way to do this, or is putting `from_expression = "select * moz-fx-data-shared-prod.telemetry.clients_daily` safe?